src/Features/ERDDAP/hooks/tabledap.ts

```diff
@@ -56,10 +56,12 @@ const erddapService = (process.env.NEXT_PUBLIC_ERDDAP_SERVICE || "https://buoyba
 
 export function urlBuilder(timeSeries: PlatformTimeSeries[], startTime?: Date, endTime?: Date): string {
   startTime = startTime ?? aWeekAgoRounded()
+  endTime = endTime ?? weeksInFuture(1)
 
   const constraints = {
     ...timeSeries[0].constraints,
     "time>=": startTime.toISOString(),
+    "time<=": endTime.toISOString(),
```

Comment:
Could we only set `time<=` when `endTime` is specified, or at least make sure it's reasonably rounded? That will allow the upstream servers to cache better.

Reply:
Yes, makes sense. Updated